alvr/server/cpp/alvr_server/OvrController.cpp

```diff
@@ -845,8 +845,9 @@ bool OvrController::onPoseUpdate(float predictionS,
             vr::VRDriverInput()->UpdateBooleanComponent(
                 m_handles[ALVR_INPUT_GRIP_TOUCH], grip > 0.7f, 0.0);
             vr::VRDriverInput()->UpdateScalarComponent(
-                m_handles[ALVR_INPUT_GRIP_FORCE], grip - 1.0, 0.0);
-            vr::VRDriverInput()->UpdateScalarComponent(m_handles[ALVR_INPUT_GRIP_VALUE], grip, 0.0);
+                m_handles[ALVR_INPUT_GRIP_FORCE], grip > 0.9f, 0.0);
```

Comment:
Here you are setting a boolean to a float component. Maybe this wasn't intended?

Reply:
You're right it wasn't, will change that.